### DISCUSSION TITLE
[Projekt] PixelIt Mini with 4 20mmx20mm LED Matrixes

### DISCUSSION THREAD

**Discussion by gumslone** [pixelit-project/PixelIt#145] [Ideas]
![IMG_2799](https://user-images.githubusercontent.com/12110353/155409479-be858c4a-ced3-487f-8548-40741208f967.jpg)
![IMG_2802](https://user-images.githubusercontent.com/12110353/155409501-5031bcef-7207-45d2-ac30-010bc3d8e9b5.jpg)

just a demo of the pixelit with tiny 4 20mmx20mm 8x8 Led Matrixes, this project is not finished yet.

[PCB Files](https://oshwlab.com/gumslone/led-matrix)

**Comment by o0shojo0o** [comment]
Oh, very nice, I'm curious about further progress :)

**Comment by Rushmed** [comment]
Nice Sir!
Whats the name of the modules?
What needs to be done to get it fully working?

**Reply by gumslone** [reply]
Hi, I've designed those modules myself, with tiny SK6805-EC15 leds, you only have to connect those four modules together and it works just like the standard matrix type 1.

**Reply by lubeda** [reply]
Very nice, do you sell these?

Sorry, i saw you are from the **ukraine**. I think there are other problemes now than selling led modules.

**Reply by foorschtbar** [reply]
Wow, thats nice! I had exactly the same idea. Also with SK6805EC15. I only wait for some time. Do u share your PCB files?

**Reply by gumslone** [reply]
> Wow, thats nice! I had exactly the same idea. Also with SK6805EC15. I only wait for some time. Do u share your PCB files?

https://oshwlab.com/gumslone/led-matrix

**Reply by foorschtbar** [reply]
Thank u! Works this without problems? Normally every LED needs a capacitors according to the datasheet?

**Reply by gumslone** [reply]
> Thank u! Works this without problems? Normally every LED needs a capacitors according to the datasheet?

Hi, unfortunately, I don't have any long-term test results, but it works for me. The capacitor is needed for better IC stability, if you have a stable power supply and add some caps on the input and output of the PCB, this will increase the stability too. But it works also without the caps.

**Comment by o0shojo0o** [comment]
Hi @gumslone, 
have you progressed with your project so that you can show something new (a housing or the like)?

**Comment by foorschtbar** [comment]
Soo, iam now a step further with my own micro matrix:

![image](https://user-images.githubusercontent.com/10727275/172069438-2cb117cd-d52f-411f-bfd3-2fe7549021d1.jpeg)

![image](https://user-images.githubusercontent.com/10727275/172069338-d31414f5-3b8f-430c-a812-5a8e9ede49ee.jpeg)

Next step is a special PCB for the back side with the ESP, because the Wemos is now larger than the matrix 🤣

The files can you find here [foorschtbar/MicroMatrix](https://github.com/foorschtbar/MicroMatrix)

**Reply by o0shojo0o** [reply]
This has also become very cool!
I'm totally happy that you created such great hardware for this project and also still shares!
My respect and recognition of the achievement.

**Reply by jekader** [reply]
Looks so cute! True spirit of opensource produces great ideas. Hope I get to have one at some point.
